alyx/data/models.py

```diff
@@ -253,6 +253,15 @@ class Dataset(BaseExperimentalData):
     md5 = models.UUIDField(blank=True, null=True,
```

Comment:
we don't delete the md5 field?

Reply:
need to do it as a 2 steps migration because cortex lab has some data in this field.